### ISSUE TITLE
Republish for PhET-iO with studio save/load

### ISSUE THREAD

**Issue by zepumph** [phetsims/gravity-force-lab#276]
This is a quarterly goal that will hopefully happen in March. @jessegreenberg can you please describe the current state of master for this sim. I know you have been working on self voicing in ISL sims. Would there need to be clean up before this occurs? Are there a11y reasons that we would want to republish this sim for a11y as well? Or perhaps we would just do a PhET-iO version of the sim.

**Comment by arouinfar** [comment]
@zepumph I've marked it as blocking sim publication, but I want to make sure you're aware this sim should also get client guides when it is republished, see #274.

**Comment by jessegreenberg** [comment]
Yes the self voicing code in the ISLC sims is in a prototype state and could use some cleanup before republishing off of master, and would be good to verify that interactive description is not disturbed. I don't think we have a goal to republish this sim for a11y reasons until the self-voicing feature is ready to publish. But that doesn't mean it can't be done, unless that makes it easier for PhET-iO. 

> This is a quarterly goal that will hopefully happen in March.

Does this mean publication is complete by end of March, and an RC needs to be ready for this sim by about end of Feb?

**Comment by zepumph** [comment]
In https://github.com/phetsims/studio/issues/203 it makes things sound like we are instead going with a maintenance release for studio save/load. That will be on hold until GAO is released.

**Comment by zepumph** [comment]
>In phetsims/studio#203 it makes things sound like we are instead going with a maintenance release for studio save/load. That will be on hold until GAO is released.

This is a lie, we are going to publish off of master. We can wait for self voicing prototype code to clean up though.
